tf/gke/gcp-users.tf

```diff
@@ -1,7 +1,6 @@
 locals {
   team_members = [
     "user:dmitchell@mozilla.com",
-    "user:jford@mozilla.com",
```

Comment:
This, too, will require a rebuild of the cluster, or perhaps project, since terraform will see it as changing the user for `team_members[1]`, `[2]`, etc. rather than just removing an element from the list.  Or something like that -- terraform and gcp get out of sync, as I recall from last time we modified this array.

Reply:
If `team_members` is only used to define `google_project_iam_member` then I don't think it will change anything other than that single user.